scoap3/articles/tests/test_article_indexing.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+import pytest
+from django.contrib.auth import get_user_model
+from django.test.client import Client
+from rest_framework.authtoken.models import Token
+
+pytestmark = pytest.mark.django_db
+User = get_user_model()
+
+
+@pytest.fixture
+def admin_user_token(user):
+    client = Client()
+    password = "admin"
+    my_admin = User.objects.create_superuser("admin", "myemail@test.com", password)
+    client.login(username=my_admin.username, password="admin")
+    user_token = Token.objects.create(user=user)
+    return {"client": client, "user_token": user_token}
+
+
+@pytest.fixture
+def license_id(admin_user_token):
+    license = {"url": "https://creativecommons.org/about/cclicenses/", "name": "cc"}
+    response = admin_user_token["client"].post(
+        "/api/license/",
+        data=license,
+        content_type="application/json",
+        HTTP_AUTHORIZATION=f"Token {admin_user_token['user_token']}",
+    )
+    return json.loads(response.content.decode("utf-8"))["id"]
+
+
+def test_article_post_and_delete(admin_user_token, license_id):
+    response = admin_user_token["client"].get(
+        "/api/license/",
```

Comment:
we should build urls from the django method, instead of adding manually - in case of future changes, same for below cases